core/parachain/pvf/run_worker.hpp

```diff
@@ -0,0 +1,113 @@
+/**
+ * Copyright Quadrivium LLC
+ * All Rights Reserved
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#pragma once
+
+#include <chrono>
+
+#include <boost/asio.hpp>
+#include <boost/process.hpp>
+#include <libp2p/basic/scheduler/asio_scheduler_backend.hpp>
+#include <libp2p/basic/scheduler/scheduler_impl.hpp>
+#include <libp2p/common/asio_buffer.hpp>
+
+#include "common/buffer.hpp"
+#include "outcome/outcome.hpp"
+
+namespace kagome::parachain {
+
+  // run(scale::encode())
+  // run(make_shared<Buffer>(scale::encode()))
+
+  template <std::invocable<outcome::result<common::Buffer>> Cb>
+  void runWorker(boost::asio::io_context &io_context,
+                 std::shared_ptr<libp2p::basic::Scheduler> scheduler,
+                 std::chrono::milliseconds timeout,
+                 const std::string &exe,
```

Comment:
Should it be filesystem::path?

Reply:
Not really needed as we don't perform any manipulations over the value and use/pass it as is.